WordPress/src/androidTest/java/org/wordpress/android/AppInitializerTest.kt

```diff
@@ -17,6 +17,7 @@ import org.wordpress.android.support.BaseTest
 class AppInitializerTest : BaseTest() {
     @Test
     fun verifyOnAppComesFromBackgroundCalled() {
+        Thread.sleep(100)
```

Comment:
This test was consistently failing without this wait when the API version was changed to a more current version, seems like updating the emulators caused a delay in some of the processes. This isn't really a UI/E2E test, the current solution to fix this is to add the shortest possible implicit wait before the assert.

Reply:
FWIW, I've [seen this test fail](https://console.firebase.google.com/u/1/project/api-project-108380595987/testlab/histories/bh.b3d1e88d8b09c8be/matrices/5406542173801078356/details?stepId=bs.1de0fc2c04a97b05&testCaseId=11) at least once on the old `Pixel 2 API 28` as well.

Reply:
> I've [seen this test fail](https://console.firebase.google.com/u/1/project/api-project-108380595987/testlab/histories/bh.b3d1e88d8b09c8be/matrices/5406542173801078356/details?stepId=bs.1de0fc2c04a97b05&testCaseId=11) at least once on the old Pixel 2 API 28 as well.

I didn't know it was flaky on `API 28`. 
`verifyOnAppComesFromBackgroundCalled()` is the only test we run on the application class before creating any activity and it does its checks just after the app launch. It seems the app's launch time is flaky on UI testing. It looks safe to me to add sleep. 👍🏻